Fixing issue 1980, action values in lower and upper case

luchoagomezt opened the pull request:
Both
    ${promise} =    Promise To    Wait For Alert    action=ACCEPT
and 
    ${promise} =    Promise To    Wait For Alert    action=accept

are valid

aaltat commented:
Hmm, this branch can not be merged to main code line because there are conflicts. Could you resolve the problem and write a test for the problem, so that similar problem does not happen again.

luchoagomezt commented:
> Hmm, this branch can not be merged to main code line because there are conflicts. Could you resolve the problem and write a test for the problem, so that similar problem does not happen again.

I fixed the conflict and added two test cases: a) action=ACCEPT and b) action=DISMISS

aaltat commented:
@allcontributors please add @luchoagomezt for code.
